src/Telemetry/readme.md

```diff
@@ -99,6 +100,20 @@ var telemetryConfig = new TelemetryManagerConfig()
 
 **Note:** The AppId must have from 3 to 20 characters and only letters and numbers are allowed.
 
+### .DefaultTelemetryConsent
+
+The default User Consent to collect telemetry (defaults to ConsentKind.OptedOut, can also be set ConsentKind.OptedIn).
```

Comment:
I have a feeling of a contradiction in the fragment 
> The default User Consent **to collect** telemetry (defaults to ConsentKind.**OptedOut**,

On the one hand it says "**to collect** by default", on the other hand it says "**OptedOut** (to NOT collect) by default".

Reply:
I'll make the text clearer.